### PR TITLE
Updated 2018 guidelines

### PR DESCRIPTION
**Pull request by therishidesai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 How to submit your project for HackIllinois 
 
-* Instructions on the process for Create are [here](https://github.com/HackIllinois/Submissions-Guidelines/tree/master/create)
-* Instructions for how to submit for Contribute are [here](https://github.com/HackIllinois/Submissions-Guidelines/tree/master/contribute)
+* If you made an Open Source project click [here](https://github.com/HackIllinois/Submissions-Guidelines/tree/master/create)
+* If you made an Open Source contribution click[here](https://github.com/HackIllinois/Submissions-Guidelines/tree/master/contribute)
```
Comment (narendasan) [inline README.md]: space needed here

File: contribute/contribute.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-##Contribute
+##If You Contributed to a Project
 
-As an attendee in the Contribute track, your submission will consist of a [devpost](https://hackillinois-2017.devpost.com/) submission and a link to a GitHub gist.
+As an attendee who made open source contributions, your submission will consist of a [devpost](https://hackillinois-2017.devpost.com/) submission and a link to a GitHub gist.
```
Comment (narendasan) [inline contribute/contribute.md]: Update the devpost link

File: create/create.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# Create
+# If You Started Something New
 
-As an attendee in the Create track, your submission should consist of a [devpost](https://hackillinois-2017.devpost.com/) submission, public code repository, as well as (at least) three specific files within that repository: *readme* (README.md), *credits* to other people (contributors.md), and *contributing guide* (contributing.md). And of course, your presence at the project expo.
+As an attendee who developed a new project, your submission should consist of a [devpost](https://hackillinois-2017.devpost.com/) submission, public code repository, as well as (at least) three specific files within that repository: *readme* (README.md), *credits* to other people (contributors.md), and *contributing guide* (contributing.md). And of course, your presence at the project expo.
```
Comment (narendasan) [inline create/create.md]: update devpost link

**Review by narendasan — CHANGES_REQUESTED** [review]

**Comment by redsn0w422** [comment]
@apache8080 are we not renaming the folders? One way to get around this is to just put all the info in one file. The problem with having folders is that it has the issue of dividing the event.

**Comment by therishidesai** [comment]
Just renamed them. I forgot about doing that on my previous commit.

**Review by redsn0w422 — CHANGES_REQUESTED** [review]
I think putting them all in one file is a good idea. Can you do that?

**Comment by aashishkapur** [comment]
@apache8080 Devpost does not exist, @redsn0w422 will create soon, update the link once it's made

**Comment by therishidesai** [comment]
@redsn0w422  So we can put it all in one file, but I don't know how to link to specific headings in markdown. Do you know how to do that?

**Comment by aashishkapur** [comment]
@apache8080 https://gist.github.com/asabaylus/3071099

**Comment by therishidesai** [comment]
One page now.

File: README.md [hunk 4]
```diff
@@ -1,10 +1,100 @@
...
+  - https://github.com/rust-lang/rust/blob/master/CONTRIBUTING.md
+  - https://github.com/redox-os/redox/blob/master/CONTRIBUTING.md
+  - https://github.com/rust-lang-nursery/rust-cookbook/blob/master/CONTRIBUTING.md
+
+##### *Bonus:*
+
+- Create comprehensive set of __issues__ to represent future features and/or current bugs
+- __Incorporate tests__ into your project
+- Provide a link to a __packaged release__
+- Proper use of source control
```
Comment (redsn0w422) [inline README.md]: Can we include examples of what good source control usage looks like?

**Review by redsn0w422 — CHANGES_REQUESTED** [review]

**Pull request merged**